hal/armv8m/nrf/91/nrf91.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system kernel
+ *
+ * nRF9160 basic peripherals control functions
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _HAL_NRF91_H_
+#define _HAL_NRF91_H_
+
+#include "../types.h"
+
+/* currently supported baud rates - 9600, 115200 */
+#define BAUD_9600   0x00275000u
+#define BAUD_115200 0x01d60000u
```

Comment:
Yeah, you are right, maybe `BAUD_115200_REGVAL` then? Here I also wanted to avoid copy-paste, but maybe it would be better to place it both in console and uart driver.